### DISCUSSION TITLE
Redis Object Cache : wp-admin caching issues

### DISCUSSION THREAD

**Discussion by ppooja-pednekar** [rhubarbgroup/.github#2] [Support (ROC)] [answered]
## Description
Redis Object Cache : wp-admin caching issues
My WP Admin section seems to be cached after configuring the plugin it i.e login page goes into loop even though I am actually logged in.
Similarly, while adding new pages/posts the id gets duplicated and hence posts are overwritten , unless i do hard refresh. Any help/guidance here will greatly help.

## Expected Behavior

Exclude cache for logged in users

## Possible Fix

Tried adding following to ignored group list, but didnt help:
    "useremail",
    "userlogins",
    "usermeta",
    "user_meta",
    "userslugs",
    "admin",
    "users",
    "blog-id-cache"

## Steps to Reproduce

1.
2.
3.
4.

## Environment

* Plugin version: Redis Object Cache (Version 2.2.3)
* PHP version:
* WordPress version: 6.2

**Comment by tillkruss** [comment]
Sorry, I don't understand what the issue is.

**Comment by ppooja-pednekar** [comment]
After configuring the Redis Object Cache , facing issues with admin section ( wp-admin ) after login.
Entire admin panel seems to have been cached. i.e login page goes into loop even though I am actually logged in.
If i manually change the path it seems to work.
Similarly, while adding new pages/posts the id gets duplicated and hence posts are overwritten , unless i do hard refresh before creating a page (which then creates a new id). 
Any help/guidance here will greatly help.

**Comment by tillkruss** [comment]
Are you using the same Redis database for 2 WordPress sites? Did you flush your cache?

**Reply by ppooja-pednekar** [reply]
No, Redis Database is not shared.  However, Have configured the Redis on WP multisite setup.  Does this need some special configurations ?
Yes, I did try clearing the cache, But it doesn't seem to help.

**Reply by tillkruss** [reply]
No, multisite does not require any extra configuration. Can you make a video of the issue?

**Reply by tillkruss** [reply]
You can post it here. Don't send anything confidential via email, nor post it here.

**Reply by tillkruss** [reply]
You'll figure it out. I believe in you.

**Reply by ppooja-pednekar** [reply]
Issue Details: https://we.tl/t-Hvu3ENg2IU

**Reply by tillkruss** [reply, marked as answer]
I'll be honest. I don't want to watch a 5min video. I'd suggest hiring a WordPress developer to assist you with caching issues on your WordPress website.

We provide support directly about Redis Object Cache.
